frontend/src/System/Status/About/About.js

```diff
@@ -30,54 +31,54 @@ class About extends Component {
     } = this.props;
 
     return (
-      <FieldSet legend="About">
+      <FieldSet legend={translate('About')}>
         <DescriptionList className={styles.descriptionList}>
           <DescriptionListItem
-            title="Version"
+            title={translate('Version')}
             data={version}
           />
 
           {
             packageVersion &&
               <DescriptionListItem
-                title="Package Version"
-                data={(packageAuthor ? <span> {packageVersion} {' by '} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
+                title={translate('PackageVersion')}
+                data={(packageAuthor ? <span> {packageVersion} {translate('by')} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
```

Comment:
Similar thing here, though this is more complicated with the HTML.

Reply:
So for this I went with translating the whole thing and if a packageAuthor exists wrapping it all in an `InlineMarkdown`.